flaml/config.py

```diff
@@ -4,7 +4,7 @@
 """
 
 N_SPLITS = 5
-RANDOM_SEED = 1
+RANDOM_SEED = 2

```

Comment:
Not going to block the PR for this, but odd we have release code with a fixed random seed

Reply:
Reset it back to its original value of 1, tests still run fine